addons/web/static/src/views/graph/graph_view.js

```diff
@@ -113,6 +114,10 @@ export class GraphView extends Component {
     setup() {
         this.actionService = useService("action");
 
+        const { arch, searchViewId, viewId } = this.props;
```

Comment:
bof bof hein

Reply:
problem: arch, type are known only in willStart of View comp or in Controller (legacy_views) --> things must be done after: in View adapter or concrete view. WithSearch comp or setupAction hook do not seem the right places to call useDebugMenu

Reply:
If one does not like the context (second parameter) used in useDebugMenu, one could create different debug items for the legacy case